code/processes/dqe.q

```diff
@@ -113,8 +113,24 @@ postback:{[runtype;idnum;proc;params;result]
     .dqe.updresultstab[runtype;idnum;.z.p;first result;result[1];`complete;params;proc]];
   }
 
+containerfn:{[funct;names;vars]
+  if[any b:where 11h=abs type each vars;
+    startvars:vars b;
+    names:names b;
+    finvars:startvars b2:where startvars in tables[];
+    names:names b2;
+    if[count finvars;
+      finvars:finvars b3:where `date in ' cols each finvars;
```

Comment:
if the goal here is to get partitioned tables, could you just compare against `.Q.pt` instead of `tables[]` above? Also has the advantage that would work for other partition schemes

Reply:
I think this & next line can be removed now that you are comparing against `.Q.pt`